front/pages/api/v1/w/[wId]/workspace-usage.ts

```diff
@@ -26,22 +26,22 @@ export const usageTables = [
 ];
 type usageTableType = (typeof usageTables)[number];
 
-const MonthSchema = t.refinement(
+const DateSchema = t.refinement(
   t.string,
-  (s): s is string => /^\d{4}-(0[1-9]|1[0-2])$/.test(s),
-  "YYYY-MM"
+  (s): s is string => /^\d{4}-(0[1-9]|1[0-2])(-([0-2]\d|3[01]))?$/.test(s),
```

Comment:
`nit`: me hating long regex, could we do something like?
```
  (s): s is string => 
    /^\d{4}-(0[1-9]|1[0-2])$/.test(s) || 
    /^\d{4}-(0[1-9]|1[0-2])-([0-2]\d|3[01])$/.test(s),
```

Reply:
Your call tbh, it's not the craziest regex either..

Reply:
Haha yes it doesn't look a lot less complicated :P